doc/README.md

```diff
@@ -1,3 +1,5 @@
 # HW1_Part2
 Version Control HW
 
+Angels and ministers of grace defend us!
```

Comment:
Dear Professor Mandli:

I found there is a typo, please fix it 

Thanks,
Jaden